src/DataFixtures/UserFixture.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures;
+
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use App\Entity\User;
+use App\Repository\User\UserRepositoryInterface;
+
+class UserFixture extends Fixture
+{
+    private $encodedPassword;
+    private $userRepository;
+
+    public function __construct(UserPasswordEncoderInterface $encodedPassword, UserRepositoryInterface $userRepository)
+    {
+        $this->encodedPassword = $encodedPassword;
+        $this->userRepository = $userRepository;
+    }
+
+    public function load(ObjectManager $manager): void
+    {
+        $user = new User(
+            $this->userRepository->nextIdentity(),
```

Comment:
As discussed with @ayumiesan (please get in touch with her for more information), you should not expose `nextIdentity` but rather create a factory method in the repository that can use the `nextIdentity` in private.